gst/nnstreamer/tensor_query/tensor_query_filter.c

```diff
@@ -0,0 +1,121 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * Copyright (C) 2021 Samsung Electronics Co., Ltd.
+ *
+ * @file    tensor_query_filter.c
+ * @date    09 Jul 2021
+ * @brief   GStreamer plugin to handle tensor query filter
+ * @author  Junhwan Kim <jejudo.kim@samsung.com>
+ * @see     http://github.com/nnstreamer/nnstreamer
+ * @bug     No known bugs
+ *
+ */
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include "tensor_query_filter.h"
+
+GST_DEBUG_CATEGORY_STATIC (gst_tensor_query_filter_debug);
+#define GST_CAT_DEFAULT gst_tensor_query_filter_debug
+
+/**
+ * @brief the capabilities of the inputs.
+ */
+static GstStaticPadTemplate sinktemplate = GST_STATIC_PAD_TEMPLATE ("sink",
+    GST_PAD_SINK,
+    GST_PAD_ALWAYS,
+    GST_STATIC_CAPS (GST_TENSORS_FLEX_CAP_DEFAULT));
```

Comment:
Tensor format will be changed later, however, `tensor_query_filter` should accept static tensor also.

Reply:
You're right. Thx